meta-resin-common/recipes-containers/balena/balena_git.bb

```diff
@@ -131,6 +132,8 @@ do_install() {
 
   install -m 0644 ${WORKDIR}/var-lib-docker.mount ${D}/${systemd_unitdir}/system
 
+  install -m 0755 ${WORKDIR}/check-balena ${D}/${bindir}/check-balena
```

Comment:
I'd name this binary `balena-healthcheck` and also move it to `/usr/lib/balena/balena-healthcheck` since this isn't meant to be a program available to everybody

Reply:
Next push will include this.